ivo-projects/todo/src/actions/index.js

```diff
@@ -1,19 +1,29 @@
-import { ADD_REMINDER, DELETE_REMINDER, CLEAR_REMINDERS } from '../constants';
+import { ADD_REMINDER, DELETE_REMINDER, CLEAR_REMINDERS, IS_EDITING_ROW } from '../actionTypes';
 
 export const addReminder = (text, dueDate) => ({
     type: ADD_REMINDER,
-    text,
-    dueDate
+    result: {
+        text,
+        dueDate
+    }
 });
 
-export const deleteReminder = (id) => {
-    const action = {
-        type: DELETE_REMINDER,
+export const deleteReminder = (id) => ({
+    type: DELETE_REMINDER,
+    result: {
         id
-    };
-    return action;
-};
+    }
+});
 
 export const clearReminders = () => ({
     type: CLEAR_REMINDERS
```

Comment:
Add result as empty object

Reply:
what do you mean by this?

Reply:
It is a good habit to have the actions having the same structure (type and result or type, result and error)